### ISSUE TITLE
Retinanet postprocessing should sigmoid the class probabilities, not softmax them

### ISSUE THREAD

**Issue by rowanz** [tensorflow/tpu#487]
https://github.com/tensorflow/tpu/blob/cab34d82a974ca4f8ced19c236462b446f0feadf/models/official/retinanet/postprocess.py#L223 seems like a bug to me, since retinanet is trained using the focal loss rather than softmax. Switching this to `tf.nn.sigmoid` gave me a ~3pt mAP improvement but YMMV.
